cpp/src/io/parquet/page_decode.cuh

```diff
@@ -487,7 +500,14 @@ __device__ void gpuDecodeStream(
             cur++;
           }
         }
-        if (cur > end || level_run <= 1) { s->error = 0x10; }
+        if (cur > end) {
+          s->set_error_code(decode_error::STREAM_OVERRUN);
```

Comment:
this is decoding RLE level data...should be RLE_STREAM_OVERRUN (in the current naming...)